php/php-rr-sdk.md

```diff
@@ -456,7 +436,7 @@ The workflow logging does not differ from any other parts of the application, th
 logs triggered while workflow is replaying. We propose to use mocked PSR-3 logger for this purposes:
 
 ```php
-$logger = new NonReplayingLogger(Workflow::ctx(), $parentLogger); // it is possible to omit ::ctx()
+$logger = Workflow::nonReplyingLogger($parentLogger);
```

Comment:
nonReplayingLogger

Maybe Workflow::getLogger?

Reply:
No, cos user must supply his own psr3 logger. Workflow does not create logger, it only proxies needed calls to the outer one.